src/test/resources/archives/caseclaimquery/modules-2/forms-0.xml

```diff
@@ -6,7 +6,9 @@
 				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/5BE25716-192B-4554-BD2A-3409ECFFACF2" uiVersion="1" version="7" name="Followup Form">
 					<name/>
 				<case xmlns="http://commcarehq.org/case/transaction/v2" case_id="" date_modified="" user_id=""/><orx:meta xmlns:cc="http://commcarehq.org/xforms"><orx:deviceID/><orx:timeStart/><orx:timeEnd/><orx:username/><orx:userID/><orx:instanceID/><cc:appVersion/><orx:drift/></orx:meta></data>
-			</instance><instance id="registry" src="jr://instance/remote"/><instance id="duplicate" src="jr://instance/remote"/>
+			</instance>
+			<instance id="registry" src="jr://instance/remote/registry"/>
+			<instance id="duplicate" src="jr://instance/remote/duplicate"/>
```

Comment:
Are these changes necessary or are you just updating them to be correct? We should at least have one test that verifies the legacy format still works.

Reply:
I see you left some in a later commit